### PR TITLE
A small function which returns the active media-query

### PR DESCRIPTION
**Pull request by TrebBrennan**

### CODE REVIEW

File: assets/js/ui-kit.js [hunk 1]
```diff
@@ -60,6 +60,16 @@ var Velocity = require('velocity-animate');
     Velocity(panel, bool ? 'slideUp' : 'slideDown', { duration: 300 });
   }
 
+  checkMedia = function(){
```
Comment (elisechant) [inline assets/js/ui-kit.js:19]: how are you planning to use this? is this a replacement for css media queries?

Comment (TrebBrennan) [reply assets/js/ui-kit.js:19]: @elisechant No it's not a replacement for css media queries.

It's just a way to check in JS which media query is currently active without having to maintain break-points in two places (JS and CSS).

I have also had an issue in the past with some browsers returning different viewport sizes in JS than the media-query it's currently using (can't remember specifically). So this uses the media-query as the source of truth.

Would use something like this:

```
      window.onresize = function(event) {
        var mediaSize = checkMedia();
        if ( mediaSize == "mobile" ) {
          // Do mobile stuff
        } else {
          // Do other stuff
        }
      };
```

Ref: http://bricss.net/post/22198838298/easily-checking-in-javascript-if-a-css-media-query

**Comment by klepas** [comment]
This seems really damn neat, if I understand it correctly.

Things left here, from what I can tell:
- tick of approval by Petch
- documentation of this feature and its application in debugging/extending the UI-Kit

**Comment by dominikwilkowski** [comment]
Looks great.
I built [something similar](https://github.com/dominikwilkowski/breakpointjs) a couple moons ago.

Not sure how the reviews go here but if I were to review this I'd say: "It's a go!"

**Review by maxious — APPROVED** [review]

**Comment by klepas** [comment]
@dominikwilkowski @maxious If we’re happy with the JS could I ask one of ya’ll to deal with the conflict for a near-future merge? ^_^

**Comment by maxious** [comment]
@klepas rebase'd

**Comment by klepas** [comment]
@maxious Danku.

Now we just need to tee up some documentation for this cc @joolswood

**Comment by joolswood** [comment]
If one of you can pair up with me sure.

**Pull request closed without merging**